valence-rs/valence
# Add minimal chat example

## Description

I've added a minimal chat example - ignoring chat signing, etc. It's in the file `crates/valence/examples/chat.rs`.

## Test Plan

Manual, unfortunately :(

Steps:
1. Open two minecraft clients, with two separate accounts (or usernames if using offline mode)
2. Connect to the server in both clients
3. Type a chat message, checking it appears to both clients, with the correct username.
4. Type a command, checking it appears only to the sender.
